tests/cli/test_rasa_interactive.py

```diff
@@ -52,15 +51,15 @@ def test_interactive_core_help(run: Callable[..., RunResult]):
 
 
 def test_pass_arguments_to_rasa_train(
-    default_stack_config: Text, monkeypatch: MonkeyPatch
+    stack_config_path, monkeypatch: MonkeyPatch
```

Comment:
can we please add type annotations for these?
```suggestion
    stack_config_path: Text, monkeypatch: MonkeyPatch
```